api/grpc/mpi/v1/common.proto

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) F5, Inc.
+//
+// This source code is licensed under the Apache License, Version 2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+
+syntax = "proto3";
+
+package f5.nginx.agent.api.grpc.mpi.v1.common;
+
+option go_package = "github.com/nginx/agent/v3/api/grpc/mpi/v1";
+
+// Meta-information associated with a message
+message MessageMeta {
+    // monotonically increasing integer
+    string message_id = 1;
+    // if 2 or more messages associated with the same workflow, use this field as an association
+    string correlation_id = 2;
+}
+
+// Represents a the status response of an command
+message CommandResponse {
+    // Command status enum
+    enum CommandStatus {
+        // Unspecified status of command
+        COMMAND_STATUS_UNSPECIFIED = 0;
```

Comment:
I'm curious why all enums have an unspecified value. Is that a common/best practice? Is it just to create a zero value?

Reply:
yes https://protobuf.dev/programming-guides/dos-donts/#unspecified-enum